### DISCUSSION TITLE
Certbot duckdns plugin in Nginx Proxy Manager LXC

### DISCUSSION THREAD

**Discussion by Depechie** [community-scripts/ProxmoxVE#1077] [Ideas]
Would be nice if the Nginx Proxy Manager LXC had the https://pypi.org/project/certbot-dns-duckdns/ pre installed.
It is needed if you want to get certificates from DuckDns.org for subdomains, like explained here https://www.youtube.com/watch?v=acturgE4TmE

**Comment by MickLesk** [comment]
Nope, we dont add something to this
